modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -73,6 +73,42 @@ GRANT READ {*} ON GRAPH * FOR (n) WHERE n.createdAt > date('2024-10-25') TO regu
 |===
 
 
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+MATCH (movie:$($label))
+----
+
+[source, cypher, role="noheader"]
+----
+CREATE (movie:$($label))
+----
+
+[source, cypher, role="noheader"]
+----
+MERGE (movie:$($label))
```

Comment:
I think it is important to show the relationship syntax as well, as that is one of the main new things about this feature (SET and REMOVE cover all node cases for dynamic labels already, so this is just syntactic sugar for them)